README.md

```diff
@@ -1 +1,58 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-world
+
+> This "Hello-world" project purpose is to practice the GitHub work flow.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://github.com/rbreva/Hello-world.git)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [OPTIONAL] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd project_name`, etc 🙃  Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.
- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.